### ISSUE TITLE
creating a debug file on all the working folders

### ISSUE THREAD

**Issue by akhilaravind** [microsoft/vscode#116307]
Issue Type: <b>Bug</b>

Afte the recent update, editor created a debug.log file in all the expanded folder(s). The content of the debug file is 
[0210/125429.205:ERROR:registration_protocol_win.cc(102)] CreateFile: The system cannot find the file specified. (0x2)
[0210/125509.441:ERROR:registration_protocol_win.cc(102)] CreateFile: The system cannot find the file specified. (0x2)

I dont understand why these files are created, and its completly annoying. I have to manually remove those files from the project.

Below is the version information

Version: 1.53.1 (system setup)
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19041

VS Code version: Code 1.53.1 (5d424b828ada08e1eb9f95d6cb41120234ef57c7, 2021-02-08T23:29:42.785Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt617cf:30253240
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30211401
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt600cf:30251589

```

</details>

**Comment by MichelCPU** [comment]
Look like an old bug coming back!  It's not the first time it append..

**Comment by Krysiek** [comment]
In my case the bug relates to TypeScript project only. No debug.log file is created when browsing pure JavaScript project written in React.

**Comment by LuigiMaestrelli** [comment]
For me, it is happening on every project, on every folder.
React-Native and pure javascript projects.

**Comment by phoshenwen** [comment]
Win10:
Debug.log is also created in an empty folder when using right click -> open with VS Code

**Comment by akhilaravind** [comment]
> In my case the bug relates to TypeScript project only. No debug.log file is created when browsing ReactJS JavaScript project.

In my case its appending to all the folder that is open

**Comment by gjsjohnmurray** [comment]
/duplicate of #115874 and will be fixed in upcoming 1.53.2
